xcell/mappers/mapper_DESY1wl.py

```diff
@@ -6,6 +6,8 @@
 
 
 class MapperDESY1wl(MapperBase):
+    map_name = 'DESY1wl'
+
```

Comment:
don't you need to update this name depending on the bin?

Reply:
That's an option, but would break the previous file naming convention. Happy to break it though. It would make things easier and cleaner

Reply:
do it!